packages/cozy-client/src/CozyClient.js

```diff
@@ -605,6 +606,28 @@ client.query(Q('io.cozy.bills'))`)
     return this.mutate(this.getDocumentSavePlan(doc), mutationOptions)
   }
 
+  /**
+   * Saves multiple documents in one batch
+   * - Can only be called with documents from the same doctype
+   * - Does not support automatic creation of references
```

Comment:
`Does not support automatic creation of references` Est-ce que ça devrait être noté dans une issue, ou est-ce un comportement normal et on ne fait ici que le préciser ?

Reply:
C'est un comportement normal à mon avis, les références sont spécifiques aux fichiers et ne sont pas utilisées pour les autres doctypes. A mon avis, il faudra créer l'issue au besoin.

Reply:
L'issue sera plus de supprimer la notion de references et de passer aux relations pour les fichiers.

Reply:
Ok merci pour l’éclaircissement 👍